.github/hacktoberfest 2021/Guidelines 2021.md

```diff
@@ -0,0 +1,45 @@
+# Guidelines for Hactoberfest 2021 ⚡

```

Comment:
Can/should we include the Open Horizon logo or an icon representation of it here?

BTW, project logo and all artwork lives under https://github.com/open-horizon/artwork